admin/src/main/resources/io/buoyant/admin/js/src/metrics_collector.js

```diff
@@ -56,8 +56,8 @@ define(['jQuery'], function($) {
     }
 
     function onAddedClients(handler) {
```

Comment:
not sure i fully grock what's going on here. i see two calls to `onAddedClients`:
1. `metricsCollector.onAddedClients(addServices)`
1. `metricsCollector.onAddedClients(addClients);`

... and the functions passed in have different signatures:
1. `function addServices(addedClients, metricsJson)`
1. `function addClients(addedClients)`

is this expected?